packages/react/src/components/header/Header.stories.tsx

```diff
@@ -190,3 +190,41 @@ export const Minimal = (args) => {
     </Header>
   );
 };
+
+export const WithSkipLink = (args) => {
+  return (
+    <Header {...args} onDidChangeLanguage={languageChangedAction}>
+      <Header.SkipLink skipTo="#content" label="Skip To Content" />
+      <Header.ActionBar
+        title="Helsingin kaupunki"
+        titleAriaLabel="Helsingin kaupunki"
+        titleUrl="https://hel.fi"
+        titleStyle={TitleStyleType.black}
+      >
+        <Header.NavigationLanguageSelector languages={languages}>
+          <h3>Tietoa muilla kielillä</h3>
+          <Link external href="www.example.com">
+            Selkosuomi
+          </Link>
+          <Link external href="www.example.com">
+            Viittomakieli
+          </Link>
+        </Header.NavigationLanguageSelector>
+
+        <Header.ActionBarItem fullWidth label="Haku" icon={IconSearch} id="action-bar-search">
+          <Header.NavigationSearch onChange={searchChangeAction} onSubmit={searchSubmitAction} label="Haku" />
+        </Header.ActionBarItem>
+        <hr style={{ order: 9 }} />
+        <Header.ActionBarItem label="Kirjaudu" icon={IconUser} style={{ order: 10 }} id="action-bar-login">
```

Comment:
Since log in isn't implemented yet, I think it might be better if you just remove this and the hr-tag on the line above so nobody thinks there is some logic already.